android/KMEA/app/src/test/java/com/tavultesoft/kmea/util/INIParserTests.java

```diff
@@ -0,0 +1,221 @@
+package com.tavultesoft.kmea.util;
+
+import org.junit.Assert;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Test;
+
+public class INIParserTests {
```

Comment:
Fun detail - Android Studio requires any desired test classes and test functions to be declared `public`.  This differs from the situations I've experienced in other IDEs.

If a test class itself is not public, Android Studio won't realize that it exists as a testing class.  Caused me quite the headache for a while.